app/journal

```diff
@@ -0,0 +1,9 @@
+libcore.io.DiskLruCache
```

Comment:
Please remove app/journal file from commit. If possible, include it on .gitignore.